.golangci.yml

```diff
@@ -59,6 +59,12 @@ linters-settings:
       # Forbid everything in syscall except the uppercase constants
       - '^syscall\.[^A-Z_]+$(# Using anything except constants from the syscall package is forbidden )?'
       - '^logrus\.Logger$'
+  depguard:
+     rules:
+      main:
+        deny:
+          - pkg: io/ioutil
+            desc: ioutil is deprecated, see https://go.dev/doc/go1.16#ioutil
```

Comment:
Why are we adding it? I checked this better the last time I suggested it and it should not be necessary. The current linters already catch it.

Reply:
Below the message from `staticcheck`, it sounds acceptable to me, so I'm for just dropping `depguard` (with the assumption that we are not benefiting from any default configuration).

```
SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.
```

Reply:
Dropped :+1: